### ISSUE TITLE
Question about Let's Encrypt

### ISSUE THREAD

**Issue by jwoertink** [nextrevision/terraform-rancher-ha-example#1]
In your example `terraform.tfvars` you have:

```
after receiving certificates from Let's Encrypt, I placed them in ./certs. modify these values with the paths to your certificates.
```

I haven't used Let's Encrypt, but the website directed me to https://certbot.eff.org/ How do I know which webserver and OS rancher uses for this, or do I just select my local OS?

**Comment by nextrevision** [comment]
That's a good question. I used the generic webserver path. This way it doesn't really matter which webserver you are running during the generation process, so long as you have one. You probably won't be using Rancher specifically for cert generation, instead Rancher just consumes the certs from the generation process (uploaded manually). It's probably best to think of the cert generation process independently of Rancher on the whole. I did a writeup of how I generated certs that I then later used with Rancher (and in this automation) that may be a helpful reference: https://thisendout.com/2016/04/21/letsencrypt-certificate-generation-with-docker/.
